Add Documentation: Why use Circuitbreaker (#3)
postmates/circuitbreaker: README.rst

    $ pip install circuitbreaker

Why not circuitfixer ? 
----------------------

> *Circuitfixer* (https://github.com/postmates/circuitfixer) is not intuitive for first time users. It uses

Reviewer: ```suggestion
*Circuitfixer* (https://github.com/postmates/circuitfixer) can be confusing for first time users. It uses
```

the idea of an exponential moving average to keep track of when the circuit breaker should open/close. In addition, each process 

> in the same pod which uses circuitfixer has an independent view of the circuitfixer state. This means that the states will 

Reviewer: Rephrase, maybe:

```
It also does not work well with multi-process applications: each process has an independent view of the circuitfixer state, so it takes much longer for all of them to detect an outage.  This is not ideal for service reliability.
```

be eventually consistent. This is not ideal if you want your service to be reliable. 

> <Drum Roll>Hello Circuitbreaker!</Drum Roll> The circuitbreaker state is shared across processes within the same pod.In addition, 

Reviewer: ```suggestion
<Drum Roll>Hello Circuitbreaker!</Drum Roll> This library uses shared memory to maintain a consistent state among processes within the same instance of an app.  In addition, 
```

> the semantics to decide when to open / close a circuit breaker has been simplified to the number of consecutive failures incurred.

Reviewer: ```suggestion
the semantics to decide when to open / close a circuit breaker are simpler, and are based on a simple counter of consecutive failures.
```

> As a first time consumer of this library, it is much more comprehnsible. These factors will help ensure maintainability, and 

Reviewer: ```suggestion
This library is meant to be much more understandable for first time users.  These factors will help ensure maintainability, and 
```
